src/python/py/models/builder.py

```diff
@@ -1939,7 +1949,7 @@ def get_args():
         "-e",
         "--execution_provider",
         required=True,
-        choices=["cpu", "cuda", "dml"],
+        choices=["cpu", "cuda", "dml", "web"],
         help="Execution provider to target with precision of model (e.g. FP16 CUDA, INT4 CPU)",
```

Comment:
could you please also update the information for web here:
https://github.com/microsoft/onnxruntime-genai/pull/426/files#diff-11d14636083c4529597f09601b8f66e5df0094978debab425bfe6dd850cfcb50R1998

Reply:
done